src/js/utils/backbone.events.ts

```diff
@@ -22,13 +51,13 @@ export default class Events {
      * @param {any} [context] - The context to apply to the callback's function invocation.
      * @returns {any} `this` context for chaining.
      */
-    on(name, callback, context) {
+    on(name: string | EventListenerDictionary, callback: EventCallback, context?: any): Events {
```

Comment:
do we not get these defs for free with the `implements EventsInterface`? i remember @zetagame saying something about this as well

Reply:
I get an ESLint `tslint:typedef` error when removing these type definitions.

Reply:
I think that's the difference with separating your class and interface (where interface has the same name) from implementing an interface. Either way, I removed `EventsInterface` as it wasn't being used.